src/tools/active_window.c

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2021 SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ **/
+
+/**
+ * This tool reads or sets the active window. It can be used for restoring back
+ * the currently active window later.
+ *
+ * Usage:
+ *
+ *   active_window [WID]
+ *
+ * Without any parameter it prints the active window ID, with a parameter
+ * it activates the window with that ID [WID].
```

Comment:
```suggestion
 * it activates and raises the window with that ID [WID].
```

Reply:
In my WM setup the focused window may be partially obscured, so I verified that `active_window` actually raises it.